hydra-tui/src/Hydra/TUI.hs

```diff
@@ -315,10 +344,9 @@ handleNewTxEvent Client{sendInput} = \case
           field = editShowableFieldWithValidate (lens id seq) "amount" (\n -> n > 0 && n <= limit)
        in newForm [field] limit
     submit s (inject . Cardano.Coin -> amount) = do
+      let tx = mkSimpleCardanoTx input (recipient, amount) (getCredentials me)
       liftIO (sendInput (NewTx tx))
       continue $ s & dialogStateL .~ NoDialog
-     where
-      tx = mkSimpleCardanoTx input (recipient, amount) (myCredentials s)
```

Comment:
One of the only situations where I prefer `let` over `where` (when there are options) -> to avoid nested `where` clauses.